docker/docker-compose.yml

```diff
@@ -48,7 +48,7 @@ services:
     ports:
       - "7070:7070"
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://arns-handover:7070/health/ping"]
+      test: ["CMD", "wget", "-q", "-O", "-", "http://localhost:7070/health/ping"]
```

Comment:
Why the switch to wget here? Using `-O -` is the default curl behaviour so I'm a bit confused by this.

Also why the switch to localhost here?

Reply:
`curl` has been added additionally to the docker image. This is not necessary, as we can use `wget` without installing anything. I will revert the change to `localhost`

Reply:
Should the same change apply to the other two uses of curl in this file?

I think this should have been in a different PR, but you could use the PR description field in future to list what the changes this PR covers are.